code/__DEFINES/subsystems.dm

```diff
@@ -237,3 +237,15 @@
 #define SSEXPLOSIONS_TURFS 2
 #define SSEXPLOSIONS_THROWS 3
 
+// Subsystem delta times or tickrates, in seconds. I.e, how many seconds in between each process() call for objects being processed by that subsystem.
+#define SSAIR_DT 0.5
+#define SSDCS_DT 1
+#define SSFASTPROCESS_DT 0.2
+#define SSMACHINES_DT 2
+#define SSMOBS_DT 2
+#define SSMOOD_DT 1
+#define SSNANITES_DT 1
+#define SSOBJ_DT 2
+#define SSPROCESSING_DT 1
+#define SSQUIRKS_DT 1
+#define SSRADIATION_DT 1
```

Comment:
change all of this to use the wait. 

```dm
#define SSAIR_DT (SSair.wait/10)
...
```

I don't like duplicating info in a way that can create a desync, or making it so nobody can change a wait without changing the same value in multiple places, and we do still need to support var edited wait times.

I might also suggest adding SECONDS to the define, to avoid confusion (such as `SSAIR_DT_SECONDS`), but don't feel like blocking on that.

Reply:
The waits have been updated to use these defines, so it's only defined in one place.

Reply:
Thats not what i asked.

The waits should remain dediseconds, both in given unit and stored unit.

Changing them to use these defines doesn't help with varedited waits. 

Change these defines to refer to the subsystem wait, keep the subsystem wait defined in the subsystem's file.

Reply:
Oh I see what you mean, then we will lose the benefit of having this available at compile time. If we're going that route I'd prefer if it was actually dynamic like we discussed with queued_time etc.

Reply:
it won't quite work as i expected, some procs and sleep and resume after fire() has ended, the calculation for those cases will be wildly off. my plan is to go through and use those defines to quickly make everything use it.

and for the reference, subsystems will exist before any object code runs. (outside of stat()), so i don't forsee an issue to just making those defines refer to the wait by global var reference.

Reply:
Alright, I'll convert the defines then